src/map-config.ts

```diff
@@ -0,0 +1,38 @@
+export type MapConfig = {
+  tileUrl: string;
+  attribution: string;
+}
+
+export const DefaultMapConfig: MapConfig = {
+  tileUrl: '//{s}.tile.jawg.io/jawg-terrain/{z}/{x}/{y}.png?access-token=t6fAKnvaPdPCucraY88YwlKjBfUHqBMvvZBIWlcp1Z9Z5FVtA02uWo6Dc9DGB2JO',
+  attribution: 'Map &copy; <a href="http://jawg.io" target="_blank" class="jawg-attrib"><b>Jawg</b>Maps</a> | Map data &copy; <a href="https://www.openstreetmap.org/copyright" target="_blank" class="osm-attrib">OpenStreetMap contributors</a>',
+};
```

Comment:
Can we create a more global config ? With configuration for new default pelias endpoint when the local storage is empty

https://github.com/pelias/compare/blob/b16b382c0c839a3ef8c0269ea4f972fd198a161a/src/views/App.vue#L392-L397

Reply:
BTW this does not work when the API is not on the root path
```js
this.hosts = createTags([`${window.location.protocol}//${window.location.host}`]); 
```

Reply:
> Can we create a more global config ? With configuration for new default pelias endpoint when the local storage is empty
> 
> https://github.com/pelias/compare/blob/b16b382c0c839a3ef8c0269ea4f972fd198a161a/src/views/App.vue#L392-L397

good idea, I'll add it.